dask/dataframe/multi.py

```diff
@@ -230,7 +230,51 @@ def require(divisions, parts, required=None):
 
 def merge_chunk(lhs, *args, **kwargs):
     empty_index_dtype = kwargs.pop("empty_index_dtype", None)
-    out = lhs.merge(*args, **kwargs)
+
+    rhs = args[0] if len(args) > 0 else None
```

Comment:
Is `rhs` ever not passed?

```suggestion
    rhs, *args = args
```

A few lines down you do `rhs.select_dtypes`, so I think it's always passed.